integration-tests/tests/run-analysis-azure.ts

```diff
@@ -105,6 +105,6 @@ const testRunAnalysisAzure = _.flowRight(
 registerTest({
   name: 'run-analysis-azure',
   fn: testRunAnalysisAzure,
-  targetEnvironments: ['dev', 'staging'],
+  targetEnvironments: ['dev'],
```

Comment:
I think we would want to disable it in general (and in that case, we don't need to change the timeout).

Reply:
PRs are all running `integration-tests-staging` - I don't think running vs dev is blocking anyone, and it'd be nice to allow the test to work if we want to invoke it deliberately. After all, the test isn't broken, the problem is that Azure VMs take forever to build and are a bit buggy on staging right now.